### ISSUE TITLE
cannot add domains includes `--` such as IDN

### ISSUE THREAD

**Issue by znz** [dokku/dokku#2318]
Description of problem:

```
vagrant@dokku:~$ dokku domains:add ruby-rails-sample xn--r8jz45g.xn--zckzah
xn--r8jz45g.xn--zckzah is invalid. exiting...
vagrant@dokku:~$ dokku domains:add ruby-rails-sample xn--eckwd4c7cu47r2wf.jp
xn--eckwd4c7cu47r2wf.jp is invalid. exiting...
```

Output of the following commands

- `dokku version`: `v0.6.5-13-g714da78`

Environment details (AWS, VirtualBox, physical, etc.): vagrant on VirtualBox

How was dokku installed?: `vagrant up --provision`

How reproducible:
`dokku domains:add` with IDN.

Actual Results:
IDN is invalid. exiting...

Expected Results:
Added IDN to app

**Comment by michaelshobbs** [comment]
Reference: https://www.icann.org/resources/pages/idn-2012-02-25-en

**Comment by josegonzalez** [comment]
Closed by #2360.
